conf/test.config

```diff
@@ -20,18 +20,18 @@ params {
     max_time   = '6.h'
 
     // Input data
-    // TODO nf-core: Specify the paths to your test data on nf-core/test-datasets
-    // TODO nf-core: Give any required params for the test so that command line flags are not needed
     input  = "${projectDir}/assets/samplesheet.csv"
 
     // Genome references
     fasta               = "https://raw.githubusercontent.com/nf-core/test-datasets/modules/data/genomics/homo_sapiens/genome/genome.fasta"
 
     // Kraken2 test db
-    kraken2db           = "https://github.com/jannikseidelQBiC/detaxizer/raw/dev/testdata/minigut_kraken.tar.gz"
+    kraken2db           = "https://github.com/nf-core/test-datasets/raw/mag/test_data/minigut_kraken.tgz"
```

Comment:
If you have decided to use that DB, probably it would be good to have that in the detaxizer branch. It might be possible that for whatever reasons that file changed in the mag branch and then the test profile wouldnt work as expected any more (i.e. fail). Doesnt need to be changed now, but before a release.